app/src/main/java/com/mapbox/maps/testapp/examples/CircleLayerClusteringActivity.kt

```diff
@@ -35,13 +35,15 @@ class CircleLayerClusteringActivity : AppCompatActivity() {
 
     mapboxMap.loadStyleUri(
       styleUri = Style.LIGHT,
-      // Disable any type of fading transition when icons collide on the map. This enhances the visual
-      // look of the data clustering together and breaking apart.
-      styleTransitionOptions = TransitionOptions.Builder()
-        .duration(0)
-        .delay(0)
-        .enablePlacementTransitions(false)
-        .build(),
+      onStylePreLoaded = {
+        // Disable any type of fading transition when icons collide on the map. This enhances the visual
+        // look of the data clustering together and breaking apart.
+        it.styleTransition = TransitionOptions.Builder()
+          .duration(0)
+          .delay(0)
+          .enablePlacementTransitions(false)
+          .build()
+      },
```

Comment:
why did we move setting style transition to this block? it work fine when specifying `styleTransitionOptions` parameter directly, that's why we did introduce that parameter

Reply:
Both the original  `MapboxMap.styleTransitionOptions` and the newly introduced `OnStylePreloaded` will work in this case, because under the hood, they have exactly exactly the same behaviour. Modified here to demonstrate this specific use case.